.github/workflows/ci.yml

```diff
@@ -50,6 +50,9 @@ jobs:
           pip install -r requirements-dev.txt  # install development dependencies and type stubs
       - name: Type check
         run: mypy --install-types --non-interactive cleanlab
+      - name: Strict type check
+        run: |
+          mypy --install-types --non-interactive --strict {cleanlab/internal/token_classification_utils.py,}
```

Comment:
The idea is to add files we know will pass the strict type-checker to this list.
It would be nice if we didn't have to write it like:
```yaml
- name: Strict type check
   run: |
     mypy --install-types --non-interactive --strict foo.py
     mypy --install-types --non-interactive --strict bar.py
     mypy --install-types --non-interactive --strict foobar.py
     ...
```
or
```yaml
- name: Strict type check
   run: |
     mypy --install-types --non-interactive --strict {foo.py,bar.py,foobar.py,...}
```

Any suggestions on how to format this, @anishathalye?

Reply:
We can revisit this in a separate PR.